src/ui/connection_profiles.ui

```diff
@@ -2297,14 +2276,28 @@ p, li { white-space: pre-wrap; }
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_3">
+              <property name="text">
+               <string>Port:</string>
+              </property>
+             </widget>
+            </item>
+            <item row="0" column="1" colspan="3">
+             <widget class="QLineEdit" name="profile_name_entry">
               <property name="font">
                <font>
                 <family>Bitstream Charter</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
+              <property name="readOnly">
```

Comment:
I think if you _frob_ a setting in the Designer it notes your tweaking and saves the revised setting - __even if it is the _normal_ one__ I think you have to hit the _left-triangle_ icon to _reset_ the setting (so that it is NOT in __bold__ in the Designer control's properties area) otherwise it gets set in this file even though it does not change it from the default setting.